src/PKSim.Core/Commands/SetExpressionLocalizationCommand.cs

```diff
@@ -0,0 +1,187 @@
+   {
+      private IndividualProtein _protein;
+      private ISimulationSubject _simulationSubject;
+      private readonly Localization _localization;
+
+      public SetExpressionLocalizationCommand(
+         IndividualProtein protein,
+         Localization localization,
+         ISimulationSubject simulationSubject,
+         IExecutionContext context)
+      {
+         _protein = protein;
+         _simulationSubject = simulationSubject;
+         _localization = localization;
+         ObjectType = PKSimConstants.ObjectTypes.Protein;
+         CommandType = PKSimConstants.Command.CommandTypeEdit;
+         context.UpdateBuildingBlockPropertiesInCommand(this, context.BuildingBlockContaining(protein));
+      }
+
+      public override void Execute(IExecutionContext context)
+      {
+         Description = PKSimConstants.Command.SetProteinTissueLocationDescription(_protein.Localization.ToString(), _localization.ToString());
+         var setLocationCommand = new SetExpressionLocalizationInProteinCommand(_protein, _localization, _simulationSubject, context).Run(context);
+         Add(setLocationCommand);
+
+         if (_localization.Is(InBloodCells))
+            AddRange(updateBloodCellsExpressionParameters(context));
+
+         if (_localization.Is(InVascularEndothelium))
+            AddRange(updateVascularEndotheliumExpressionParameters(context));
+
+         if (_localization.Is(InTissue))
+            AddRange(updateTissueExpressionParameters(context));
+
+         _protein = null;
+         _simulationSubject = null;
+
+         //update properties from first command
+         this.UpdatePropertiesFrom(setLocationCommand);
+      }
+
+      private IEnumerable<IOSPSuiteCommand> updateTissueExpressionParameters(IExecutionContext context)
+      {
+         var command = new PKSimMacroCommand();
+         var parameterTask = context.Resolve<IParameterTask>();
+         //We need to iterate over all parameters defined in the protein and update the values as expected
+         var allInterstitialFractionParameters = _simulationSubject.Individual.GetAllChildren<IParameter>(x =>
+            x.IsNamed(CoreConstants.Parameters.FRACTION_EXPRESSED_INTERSTITIAL) && x.ParentContainer.IsNamed(_protein.Name));
+
+         var allTissueRelExpParameters = _simulationSubject.Individual.GetAllChildren<IParameter>(x =>
+            x.IsNamed(CoreConstants.Parameters.REL_EXP) && x.ParentContainer.IsNamed(_protein.Name));
+
+
+         command.AddRange(setParametersForFlags(parameterTask, None, InTissue, allTissueRelExpParameters.Select(x => (x, 0.0)).ToArray()));
+         command.AddRange(setParametersForFlags(parameterTask, Interstitial, Intracellular, allInterstitialFractionParameters.Select(x => (x, 1.0)).ToArray()));
+         command.AddRange(setParametersForFlags(parameterTask, Intracellular, Interstitial, allInterstitialFractionParameters.Select(x => (x, 0.0)).ToArray()));
+
```

Comment:
should we add here and in other functions a comment that no action is required when _all localization settings of a group_ are active?
(thus explaining e.g. why the combination {Enabled="InTissue", "Disabled"=None} is not required here)?

Reply:
sure. Tests needs to be written anyways but yes adding